src/models/validators/CommonSchema.ts

```diff
@@ -1,6 +1,6 @@
 import * as Joi from 'joi';
-import { ValidationErrorItem } from 'joi';
-import { TESTING_ERRORS, VEHICLE_TYPES } from '../../assets/Enums';
+import { TestStationTypes } from "@dvsa/cvs-type-definitions/types/v1/enums/testStationType.enum";
+import { VEHICLE_TYPES } from '../../assets/Enums';
```

Comment:
Do we want to change this to use the VEHICLE_TYPES from type def too?

Reply:
I think as there is a ticket already in the backlog to use the centrally defined type-definitions, we can probably leave it for now. For the purpose of getting this VEF work across the line just ensuring testStationTypes is updated should be sufficient IMO.